.github/workflows/update-aaguids.yml

```diff
@@ -0,0 +1,36 @@
+name: Update AAGUIDs
+
+on:
+  schedule:
+    - cron: "0 0 * * 0" # Runs every Sunday at midnight
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  update-aaguids:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install jq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
```

Comment:
jq should be pre-installed on the Actions runtime: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md